src/main/java/org/springframework/social/bitbucket/api/EmailsOperations.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (C) 2012 Eric Bottard / Guillaume Lederrey (eric.bottard+ghpublic@gmail.com / guillaume.lederrey@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.social.bitbucket.api;
+
+import java.util.List;
+
+/**
+ * emails Resource
+ * @see "https://confluence.atlassian.com/display/BITBUCKET/emails+Resource"
+ *
+ * @author Cyprian Śniegota
+ * @since 2.0.0
+ */
+public interface EmailsOperations {
+
+    /**
+     * Gets the email addresses associated with the account.
+     *
+     * @param accountName The name of an individual or team account.
+     * @return list of email addresses
+     */
+    List<BitBucketEmailAddress> getListOfUserEmailAddresses(String accountName);
```

Comment:
This name refers to the name of command described in docs: https://confluence.atlassian.com/display/BITBUCKET/emails+Resource#emailsResource-GETalistofuser%27semailaddresses (it should be getAListOfUserEmailAddresses)